Extensions/PrimitiveDrawing/Extension.cpp

```diff
@@ -366,6 +366,16 @@ void DeclarePrimitiveDrawingExtension(gd::PlatformExtension& extension) {
       .SetFunctionName("IsClearedBetweenFrames")
       .SetIncludeFile("PrimitiveDrawing/ShapePainterObject.h");
 
+  obj.AddExpression(
+        "IsClearBetweenFrames",
+        _("Is clear between frames"),
+        _("Return true if the object is clear between two frames."),
```

Comment:
Everywhere: "Is cleared between frames" ("ed" to Clear)

Also, avoid "Return true" which is a very "programming language" wording. Instead, use "Check": 
"Check if the object is cleared between each frame."